include/fc/array.hpp

```diff
@@ -113,9 +113,11 @@ namespace fc {
     if( ve.size() )
     {
         memcpy(&bi, ve.data(), fc::min<size_t>(ve.size(),sizeof(bi)) );
+    }else{
+        for(size_t i=0;i<N;i++){
+            bi.data[i]=T();
```

Comment:
This is a functional change, zeroing out something is not necessarily the same as invoking its no-arg constructor.
If the compile complains about memset'ing a class but not memcpy'ing it, memset an array of appropriate size and memcpy that. Still less efficient than the original though, anyone got a better idea?

Reply:
Zero initialization may not always equals setting memory to zero, 
and [here](https://en.cppreference.com/w/cpp/language/zero_initialization) is a reference

Reply:
Yes, that's what I said.